pizza_cutter/des_pizza_cutter/tests/test_se_image_psf.py

```diff
@@ -0,0 +1,150 @@
+import numpy as np
```

Comment:
This is the file of tests for the PSF center. I am trying a bunch of different offsets and making sure the PSF center comes out in the right spot.